iohk-monitoring/src/Cardano/BM/Snocket.hs

```diff
@@ -29,19 +29,13 @@ module Cardano.BM.Snocket
 
 import           Control.Exception
 import           Control.Monad (when)
-import           Network.Socket (
-#if !defined(mingw32_HOST_OS)
-                                  Family (AF_UNIX),
-#endif
-                                  Socket
-                                , SockAddr (..)
-                                )
+import           Network.Socket (Family (AF_UNIX), SockAddr (..), Socket)
 import qualified Network.Socket as Socket
 #if defined(mingw32_HOST_OS)
 import           Data.Bits
-import qualified System.Win32            as Win32
+import qualified System.Win32 as Win32
+import qualified System.Win32.Async as Win32.Async
```

Comment:
This feels like a regression -- previously aliases were aligned vertically, which provided a table-like feeling.